django/contrib/admin/templates/admin/base.html

```diff
@@ -20,6 +20,97 @@
 </head>
 {% load i18n %}
 
+<style>
+html {
```

Comment:
The convention is to use `:root` for global variables.

```suggestion
:root {
```

Reply:
Good catch, thanks!